README.md

```diff
@@ -4,3 +4,39 @@ Hoa is a **modular**, **extensible** and **structured** set of PHP libraries.
 Moreover, Hoa aims at being a bridge between industrial and research worlds.
 
 # Hoathis\SymfonyConsoleBundle [![Build Status](https://travis-ci.org/hoaproject/Contributions-Symfony-ConsoleBundle.png?branch=master)](https://travis-ci.org/hoaproject/Contributions-Symfony-ConsoleBundle)
+
+## Installation
+
+Add these lines to your `require` section:
+
+```json
+{
+    "require": {
+        "hoathis/symfony-console-bundle": "~1.0"
+    }
+}
+```
+
+Then install dependencies:
+
+```sh
+$ composer update hoathis/symfony-console-bundle
+```
```

Comment:
The issue was opened ;)

I'll also update the README in https://github.com/hoaproject/Contributions-Symfony-ConsoleBridge with this generic template.